target_woocommerce/sinks.py

```diff
@@ -290,7 +292,7 @@ def process_record(self, record: dict, context: dict) -> None:
         response = self.request_api("PUT", endpoint, request_data=record)
         product_response = response.json()
         id = product_response.get("id")
-        self.logger.info(f"{self.name} updated for id: {id}")
+        self.logger.info(f"{self.name} updated for id: {id}, new stock: {record.get('stock_quantity')}.")
```

Comment:
is `record.get('stock_quantity')` going to be correct? wouldn't we need to use the `product_response` to get this?

Reply:
log fixed